sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -625,28 +650,81 @@ ze_result_t ZeCall::doCall(ze_result_t ZeResult, const char *ZeName,
   if (!(condition))                                                            \
     return error;
 
+// Returns bool value indicating whether queue supports in-order optimization
+// for provided type of event.
+bool _pi_queue::supportsInOrderQueueOptimization(bool HostVisible,
+                                                 bool IsDiscarded) {
```

Comment:
There can be multiple different optimization of in-order queues, so maybe call this one more specifically?

Reply:
Renamed to be more specific.